<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/logo</normaloff>:/icons/logo</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Control</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="replotButton">
           <property name="text">
            <string>Replot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoomButton">
           <property name="text">
            <string>Zoom</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoomAllButton">
           <property name="text">
            <string>Zoom all</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="networkGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Network</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QGridLayout" name="networkLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="startStopButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Start/stop network exchange</string>
             </property>
             <property name="statusTip">
              <string>Start/stop network exchange</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/refresh</normaloff>:/icons/refresh</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="monitorGroupBox">
        <property name="title">
         <string>Monitor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="monitorPointCountSpinBox">
           <property name="suffix">
            <string> points</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="monitorOnOffCheckBox">
           <property name="text">
            <string>On/Off</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="powerGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Power</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="powerLimitLabel">
           <property name="toolTip">
            <string>Power limit</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QProgressBar" name="powerBar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>12</red>
                   <green>9</green>
                   <blue>191</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>12</red>
                   <green>9</green>
                   <blue>191</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>86</red>
                   <green>117</green>
                   <blue>148</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="textDirection">
              <enum>QProgressBar::TopToBottom</enum>
             </property>
             <property name="format">
              <string>%p</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="powerPercentLabel">
             <property name="toolTip">
              <string>Power percentage</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="powerValueLabel">
           <property name="toolTip">
            <string>Actual power</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="rxLabel">
        <property name="text">
         <string>Rx: 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="txLabel">
        <property name="text">
         <string>Tx: 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="rttLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="numPointsLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="numTransitionPointsLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_recent"/>
    <addaction name="actionBrowse_network"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionExperiment_control"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionASCII_data"/>
    <addaction name="actionParsed_data"/>
    <addaction name="actionTable_data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/saveas</normaloff>:/icons/saveas</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBrowse_network">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/internet</normaloff>:/icons/internet</iconset>
   </property>
   <property name="text">
    <string>Browse network</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionExperiment_control">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/experiment</normaloff>:/icons/experiment</iconset>
   </property>
   <property name="text">
    <string>Experiment control</string>
   </property>
  </action>
  <action name="actionASCII_data">
   <property name="text">
    <string>ASCII data</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionParsed_data">
   <property name="text">
    <string>Parsed data</string>
   </property>
  </action>
  <action name="actionTable_data">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionOpen_recent">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
   </property>
   <property name="text">
    <string>Open recent</string>
   </property>
   <property name="statusTip">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
